src/lib.rs

```diff
@@ -35,16 +36,19 @@
 //!
 //! - `serde`: Enable serialization/deserialization via [serde].
 //! - `rkyv`: Deprecated, use the `rkyv-*` features.
-//! - `rkyv-16`: Enable serialization/deserialization via [rkyv], using 16-bit integers for integral `*size` types.
-//! - `rkyv-32`: Enable serialization/deserialization via [rkyv], using 32-bit integers for integral `*size` types.
-//! - `rkyv-64`: Enable serialization/deserialization via [rkyv], using 64-bit integers for integral `*size` types.
+//! - `rkyv-16`: Enable serialization/deserialization via [rkyv],
```

Comment:
In particular, why use the shorter lines here?

Reply:
The alternative would be
```rust
//! - `rkyv-16`: Enable serialization/deserialization via [rkyv], using 16-bit integers for integral
//!   `*size` types.
```

A manual wrap in this case seemed a little nicer.

Reply:
Fair enough.